CONTRIBUTING.md

```diff
@@ -10,6 +10,21 @@ using [Azure/autorest.go][] and [Azure/autorest][]. These generated packages dep
 
 For bugs or feature requests you can submit them using the [Github issues page][issues] and filling the appropriate template.
 
+## Building
+
+SDKs are either "track 1" or "track 2":
+
+- Track 1 SDKs are found in the services/ and profiles/ top level folders. 
+    - CI is in \azure-pipelines.yml
+- Track 2 SDK are foudn in the sdk/ top level folder.
+    - CI is in \eng\pipelines\templates\steps\build.yml
```

Comment:
Nit:  I tend to universally use `/` path separator chars as they work on all OS.